tools/VscPreviewExtension/media/js/htmlUpdate.js

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
+$(function () {
+  var port = $("meta[name='port']")[0].content;
+  var pageRefreshFunctionName = "refresh";
+  var filePath = $("meta[name='filePath']")[0].content;
+  var lastLocation = 0;
+  var filePathEscape = filePath.replace(/\\/g, "\\\\");
+  var rightClick = false;
```

Comment:
when the user uses `match from right to left` by clicking the right side, we want the behavior is that the left side navigates to the corresponding selection, but right side should not have any change, but without this status, the right side will navigate to the place which has been clicked.

---
In reply to: [113861663](https://github.com/dotnet/docfx/pull/1612#discussion_r113861663) [](ancestors = 113861663)